legal-api/src/legal_api/resources/v2/business/business_filings/business_filings.py

```diff
@@ -927,19 +927,23 @@ def create_invoice(business: Business,  # pylint: disable=too-many-locals,too-ma
             'businessInfo': {
                 'businessIdentifier': f'{business.identifier}',
                 'corpType': f'{corp_type}',
-                'businessName': f'{business.legal_name}',
-                'contactInfo': {'city': mailing_address.city,
-                                'postalCode': mailing_address.postal_code,
-                                'province': mailing_address.region,
-                                'addressLine1': mailing_address.street,
-                                'country': mailing_address.country}
+                'businessName': f'{business.legal_name}'
             },
             'filingInfo': {
                 'filingIdentifier': f'{filing.id}',
                 'filingTypes': filing_types
             },
             'details': ListFilingResource.details_for_invoice(business.identifier, corp_type)
         }
+
+        if mailing_address:
+            payload['businessInfo']['contactInfo'] = {
+                'city': mailing_address.city,
+                'postalCode': mailing_address.postal_code,
+                'province': mailing_address.region,
+                'addressLine1': mailing_address.street,
+                'country': mailing_address.country
+            }
```

Comment:
Where does this end up -- are there any downstream impacts? Will this data be saved in the db? Will the Filer see this?

And, how do we test this?

Reply:
This change won't impact anywhere else. This payload is specific to pay-api call.